src/main/web_src/src/components/fagsystem/krrstub/visning/KrrVisning.js

```diff
@@ -10,6 +10,7 @@ export const Visning = ({ data }) => {
 		<>
 			<TitleValue title="Mobilnummer" value={data.mobil} />
 			<TitleValue title="E-post" value={data.epost} />
+			<TitleValue title="Språk" value={data.spraak} />
```

Comment:
Sett `value={Formatters.showLabel('spraaktype', data.spraak)}` her også (som i BestillingKriterieMapper) for å vise label istedenfor value :)